src/rxretries.js

```diff
@@ -18,5 +18,4 @@ const mapped = arrO.pipe(
 )
 
 mapped.subscribe({
-  next: v => console.log(v)
```

Comment:
Import logger `const Logger = require('@mojaloop/central-services-shared').Logger`, and change `console.log(v)` ---> `Logger.debug(v)`